lib/generators/simple_abs/templates/migration.rb

```diff
@@ -1,19 +1,16 @@
-class CreateAlternativesTable < ActiveRecord::Migration
+class CreateAbTest < ActiveRecord::Migration
```

Comment:
It suggests to me that there is name-matching between lib/generators/simple_abs/simple_abs_generator.rb:20 and this table name. They matched in the base branch but not here. Can you confirm the migration template works?